plugins/dialog/src/desktop.rs

```diff
@@ -115,6 +115,12 @@ impl From<MessageDialogButtons> for rfd::MessageButtons {
             MessageDialogButtons::YesNo => Self::YesNo,
             MessageDialogButtons::OkCustom(ok) => Self::OkCustom(ok),
             MessageDialogButtons::OkCancelCustom(ok, cancel) => Self::OkCancelCustom(ok, cancel),
+            MessageDialogButtons::YesNoCancel => {
+                Self::YesNoCancelCustom(YES.into(), NO.into(), CANCEL.into())
+            }
```

Comment:
```suggestion
            MessageDialogButtons::YesNoCancel => Self::YesNoCancel,
```

Reply:
rfd doesn't have this variant

Reply:
Nvm, it actually has it, I must have been blind